### ISSUE TITLE
Fatal Crash on v2.0.2 - Rename mime.lookup() to mime.getType()

### ISSUE THREAD

**Issue by dscalzi** [bowheart/ejs-electron#11]
There may have been some incompatible mime changes since the dependency upgrade.

On startup, getting this issue:

```shell
TypeError: mime.lookup is not a function
    at Function.protocolListener (\node_modules\ejs-electron\index.js:118:29)
```

@bowheart Looks like changing mime.lookup() to mime.getType() would fix it.

See more:
https://www.npmjs.com/package/mime

**Comment by bowheart** [comment]
Ouch! Man, do I need to create tests for this little library? :dizzy_face: That's what I get for trying to Open Source in the middle of work.

Thanks!

**Comment by bowheart** [comment]
Fix now available on `ejs-electron@2.0.3`.
